core/controller/src/main/scala/whisk/core/loadBalancer/LoadBalancerService.scala

```diff
@@ -84,15 +89,23 @@ trait LoadBalancer {
   def publish(action: ExecutableWhiskActionMetaData, msg: ActivationMessage)(
     implicit transid: TransactionId): Future[Future[Either[ActivationId, WhiskActivation]]]
 
+  /**
+   * Return a message indicating the health of the containers and/or container pool in general
+   * @return a Future[String] representing the heal response that will be sent to the client
```

Comment:
`Future[String]` -> `Future[JsObject]`
`the heal response that will be sent to the client` -> the health of the container pools managed by the load balancer.

We probably want to raise the JsObject into some abstract type e.g., `(InstanceId, InvokerState)` used below, that at least has the name of the resource, it's heath (up/down) vs the opaque JsValue now.